sockets/color_list.py

```diff
@@ -0,0 +1,15 @@
+import bpy
+from .. base_types.socket import AnimationNodeSocket
+
+class ColorListSocket(bpy.types.NodeSocket, AnimationNodeSocket):
+    bl_idname = "an_ColorListSocket"
+    bl_label = "Color List Socket"
+    dataType = "Color List"
+    allowedInputTypes = ["Color List"]
+    drawColor = (0.8, 0.8, 0.2, 0.5)
+
+    def getValueCode(self):
+        return "[]"
+
+    def getCopyExpression(self):
+        return "[element.copy() for element in value]"
```

Comment:
a color is just a list with 4 elements and does not have a copy function. Please use this expression:
[element[:] for element in value]